src/MPL-1.1.xml

```diff
@@ -0,0 +1,553 @@
+               as a series of files, a Modification is: 
+              <br/>Any addition to or deletion from the contents of a file containing Original Code or
+                   previous Modifications. 
+              <br/>Any new file that contains any part of the Original Code or previous Modifications. 
+            </p>
+          </li>
+          <li>
+            <b>1.10.</b>
+            <p>&quot;Original Code&quot; means Source Code of computer software code which is described in
+               the Source Code notice required by Exhibit A as Original Code, and which, at the time of
+               its release under this License is not already Covered Code governed by this License.</p>
+            <p>1.10.1. &quot;Patent Claims&quot; means any patent claim(s), now owned or hereafter acquired,
+               including without limitation, method, process, and apparatus claims, in any patent
+               Licensable by grantor.</p>
+          </li>
+          <li>
+            <b>1.11.</b>
+            <p>&quot;Source Code&quot; means the preferred form of the Covered Code for making modifications
+               to it, including all modules it contains, plus any associated interface definition files,
+               scripts used to control compilation and installation of an Executable, or source code
+               differential comparisons against either the Original Code or another well known, available
+               Covered Code of the Contributor&apos;s choice. The Source Code can be in a compressed or
+               archival form, provided the appropriate decompression or de-archiving software is widely
+               available for no charge.</p>
+          </li>
+          <li>
+            <b>1.12.</b>
+            <p>&quot;You&quot; (or &quot;Your&quot;) means an individual or a legal entity exercising rights
+               under, and complying with all of the terms of, this License or a future version of this
+               License issued under Section 6.1. For legal entities, &quot;You&quot; includes any entity
+               which controls, is controlled by, or is under common control with You. For purposes of
+               this definition, &quot;control&quot; means (a) the power, direct or indirect, to cause the
+               direction or management of such entity, whether by contract or otherwise, or (b) ownership
+               of more than fifty percent (50%) of the outstanding shares or beneficial ownership of such
+               entity.</p>
+          </li>
+        </list>
+        <li>
+          <b>2.</b>
+          <p>Source Code License.</p>
+        </li>
+        <list>
+          <li>
+            <b>2.1.</b>
+            <p>The Initial Developer Grant. The Initial Developer hereby grants You a world-wide,
+               royalty-free, non-exclusive license, subject to third party intellectual property
+               claims:</p>
+          </li>
+          <list>
```

Comment:
This list should be a child of the previous list. Similar list indentation issues exist throughout file. Pull request submitted.